src/main/java/edu/tamu/app/controller/RemoteProjectController.java

```diff
@@ -0,0 +1,77 @@
+package edu.tamu.app.controller;
+
+import static edu.tamu.weaver.response.ApiStatus.ERROR;
+import static edu.tamu.weaver.response.ApiStatus.SUCCESS;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.tamu.app.cache.model.RemoteProject;
+import edu.tamu.app.model.Product;
+import edu.tamu.app.model.RemoteProjectInfo;
+import edu.tamu.app.model.RemoteProjectManager;
+import edu.tamu.app.model.repo.ProductRepo;
+import edu.tamu.app.service.manager.RemoteProjectManagerBean;
+import edu.tamu.app.service.registry.ManagementBeanRegistry;
+import edu.tamu.weaver.response.ApiResponse;
+
+@RestController
+@RequestMapping("/projects/remote")
```

Comment:
Should be /remote-projects

Reply:
That is actually not something new.
It has been `/projects/remote` since [2018](https://github.com/TAMULib/ProjectManagementService/blame/65b33fbec8394f0ffd94cba06d6915bafc48277e/src/main/java/edu/tamu/app/cache/controller/RemoteProductsCacheController.java).
Are you suggesting that we change the behavior?

Also, that referenced PR seems to have changes to the documentation.
I think I need to update the documentation for the current endpoints.